src/scss/_brand.scss

```diff
@@ -0,0 +1,141 @@
+////
+/// @group o-label
```

Comment:
`@group [component name]` is redundant (registry ui assigns a default group name). In the future we might want to use `@group` to categories mixins within a component though.

Reply:
Sorry, I'm being pedantic though 😅